_user/rotrectangle/Makefile

```diff
@@ -0,0 +1,13 @@
+#
+# Makefile for user application
+#
+# Copyright 2021 Phoenix Systems
+#
+
+NAME := rotrectangle
+LOCAL_SRCS := main.c
+LIBS := libgraph libvga libvirtio
+
+ifneq ($(TARGET_FAMILY), host-pc)
+	include $(binary.mk)
```

Comment:
use spaces for indentation in Makefiles outside of receipes.

I don't think `_user` folder would ever be built for `host-pc` target (if it somehow is - it's a build error and should be fixed elsewhere), so we don't need to guard against it.